### PR TITLE
[ADD] Website: features blocks

### PR DESCRIPTION
**Pull request by Donapi**
taskid-3733828

### CODE REVIEW

File: content/applications/websites/website/web_design/building_blocks/features_blocks.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+===============
+Features blocks
+===============
+
+Use the :guilabel:`Features` blocks to add advanced functionalities to your website, such as
+describing the attributes of a product or service. The blocks from this category allow you to
+present the content of your website in an illustrative method rather than sticking to the textual
+content. This documentation presents the :guilabel:`Table of content` building block.
```
Comment (chiaraprattico) [inline content/applications/websites/website/web_design/building_blocks/features_blocks.rst:8]: This documentation presents the :guilabel:`Table of content` building block.


Presenting it this way it's not typical documentation style. Instead I would say: among different feature blocks, there is the :guilabel:`Table of content` building block --> Explain a little what it does and why we mention it here (one line)

File: content/applications/websites/website/web_design/building_blocks/features_blocks.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+.. image::features_blocks/features_blocks.png
+   :alt: The features blocks available in the website builder.
+
+.. seealso::
+   :doc:`Building blocks <../building_blocks>`
+
+Table of content
+================
+
+You can add a :guilabel:`Table of content` building block to your website by dragging and dropping
```
Comment (chiaraprattico) [inline content/applications/websites/website/web_design/building_blocks/features_blocks.rst:19]: I would link this page here so they can know the more general setting on how to add blocks https://runbot137.odoo.com/runbot/static/build/59908338-17-0/logs/build/html/applications/websites/website/web_design/building_blocks.html

File: content/applications/websites/website/web_design/building_blocks/features_blocks.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+it from the :guilabel:`Features` blocks section in the website configurator.
+
+Once added, you can customize the appearance and functionality of the table of contents.
+
+Header 2 titles automatically appear on the left of your page, helping users navigate your content
+more efficiently.
+
+.. image:: features_blocks/table_of_content.png
+   :alt: Displaying the table of content building block
```
Comment (chiaraprattico) [inline content/applications/websites/website/web_design/building_blocks/features_blocks.rst:28]: do you think we need to see the whole screen, maybe the edit options on the right are not useful, not sure, but I feel like it's not focusing on the H titles on the left

**Review by chiaraprattico — CHANGES_REQUESTED** [review]
Hi @Donapi it seems like you have a conflict, the branch might not be up to date. 

image

**Review by Donapi — COMMENTED** [review]
Thanks for your review, let's discuss IRL if you want :)

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**